python-lib/tox.ini

```diff
@@ -0,0 +1,51 @@
+[tox]
+envlist = py27,py35
+
+[base]
+deps =
+    nose==1.3.0
+    rednose==0.4.1
+    coverage==3.6
+    pep8==1.4.3
+    jinja2==2.6
+    mock==1.0.1
+    requests
+    requests_mock
+
+[testenv]
+
+commands =
+    coverage erase
+    coverage run --branch --source {toxinidir}/balrogclient {envbindir}/nosetests -v --with-xunit --rednose --force-color {toxinidir}/balrogclient {posargs}
+
+
+[testenv:py27]
+basepython = python2.7
```

Comment:
Yay, 2.7 and 3.5! We may want to run against 2.6 as well, if we still use that in production anywhere...I'm trying to find out if that's the case or not.